README.md

```diff
@@ -45,7 +45,8 @@ This builds the plugin in `/build/rootfs`
 
 ###  Task manifest
 User need to provide following parameters in configuration for publisher:
-- `address` -  IP address with port number for RabbitMQ host,
+- `uri` -  AMQP URI scheme for connecting to server. URI supports the following form: "user:pass@ip:port/vhost".
+           User, password, and vhost are optional. When not specified, the default values for RabbitMQ will be used.
```

Comment:
@geauxvirtual, thanks for your update. Is there any reason you don't want to mention "guest:guest" and the default virtual host "/"?